lib/src/client_impl.dart

```diff
@@ -65,19 +65,19 @@ class MeiliSearchClientImpl implements MeiliSearchClient {
   }
 
   @override
-  Future<Map<String, dynamic>> getRawIndex(String uid) async {
+  Future<Map<String, Object?>> getRawIndex(String uid) async {
```

Comment:
yes, this could be considered breaking, since `dynamic` just disables all type checking and analysis, but the migration guide is to just use `as` whenever an `Object?` is involved, which is the whole point of explicit casting